ppinot-model/src/test/java/es/us/isa/ppinot/evaluation/computers/TimeMeasureComputerTest.java

```diff
@@ -52,10 +51,48 @@ public void testComputeLinearInstancesWithSeveralFrom() throws Exception {
         asserter.assertInstanceHasDoubleValue("i1", 30);
     }
 
+    @Test
+    public void testComputeLinearInstancesWithSeveralTo() throws Exception {
```

Comment:
Besides this test, which is START (1), END (1), END (2) you should add a new test that checks that in this sequence: START (1), END (1), START (2), END (2),  and in this sequence: START (1), START (2), END (1), END (2) it evaluates as START(1) -> END (1)

Reply:
Ok.